### PR TITLE
fix/TEN-12-eventloop-blocking-tests

### PR DESCRIPTION
**Pull request by southbite**

### CODE REVIEW

File: packages/happn-cluster/lib/services/orchestrator/constants.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  STATES: {
+    CONNECTING: 'CONNECTING',
+    SUBSCRIBING: 'SUBSCRIBING',
+    STABLE: 'STABLE',
+    ISOLATED: 'ISOLATED',
+    CONNECTING_INSUFFICIENT_PEERS: 'CONNECTING-INSUFFICIENT PEERS',
```
Comment (southbite) [inline packages/happn-cluster/lib/services/orchestrator/constants.js:7]: why is this inconsistent? - and then space

File: packages/happn-cluster/RELEASES.md [hunk 2]
```diff
@@ -268,6 +268,11 @@ non-release 2017-12-14
 -----------------
   - TEN-114: happner-2: enforce MESH_GST, MESH_ADMIN upsert on startup, ensures db dir
 
+<<<<<<< HEAD
```
Comment (southbite) [inline packages/happn-cluster/RELEASES.md:271]: resolve conflict

File: packages/happn-cluster/test/integration/05-func-orchestrator-stabilise-simultaneous-depart.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+        ).pop();
+      else {
+        config = (
+          await testUtils.createMultiServiceMemberConfigs(
+            testSequence,
+            clusterSize,
+            false,
+            false,
+            {
+              // membership: {
```
Comment (southbite) [inline packages/happn-cluster/test/integration/05-func-orchestrator-stabilise-simultaneous-depart.js:64]: dead code

File: packages/happn-cluster/test/integration/05-func-orchestrator-stabilise-simultaneous-depart.js [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+      //
+      // so it gets ECONNREFUSED
+      //
+      // safe at this point to assume the member is not there (literally no socket listening)
+      // so the member is removed from the test.expectation list of members-that-must-be-connected-to-for-stabilise
+      let config;
+      if (!testConfig.clusterConfig)
+        config = (
+          await testUtils.createMemberConfigs(testSequence, clusterSize + 1, false, false, {
+            // membership: {
```
Comment (southbite) [inline packages/happn-cluster/test/integration/05-func-orchestrator-stabilise-simultaneous-depart.js:51]: dead code

File: packages/happn-cluster/lib/services/orchestrator/constants.js (hunk 1 above)
Comment (JancoHY) [reply packages/happn-cluster/lib/services/orchestrator/constants.js:7]: Was intended to be 'human readable', should probably have been "CONNECTING - INSUFFICIENT PEERS"
will make it consistent

**Review by southbite — COMMENTED** [review]
update versions please

**Pull request closed without merging**